### PR TITLE
Allow custom class via Class parameter

### PR DESCRIPTION
**Pull request by zbecknell**
It looks like some sort of white space changes occurred in the code behind!  The only actual change there is at line 54.

I'm using the library and needed a custom class applied to the input, so this is a nice easy change.

**Edit:** Fixed whitespace changes.

### CODE REVIEW

File: src/Blazored.Typeahead/BlazoredTypeahead.razor.cs [hunk 1]
```diff
@@ -1,120 +1,121 @@
...
+        [Parameter] public RenderFragment FooterTemplate { get; set; }
+
+        [Parameter(CaptureUnmatchedValues = true)] public IReadOnlyDictionary<string, object> AdditionalAttributes { get; set; }
+        [Parameter] public int MinimumLength { get; set; } = 1;
+        [Parameter] public int Debounce { get; set; } = 300;
+        [Parameter] public int MaximumSuggestions { get; set; } = 10;
+        [Parameter] public bool Disabled { get; set; } = false;
+        [Parameter] public bool EnableDropDown { get; set; } = false;
+        [Parameter] public bool ShowDropDownOnFocus { get; set; } = false;
+        [Parameter] public string Class { get; set; } = "";
```
Comment (zbecknell) [inline src/Blazored.Typeahead/BlazoredTypeahead.razor.cs]: Here's the actual change. I hate that the diff sucks!

Comment (zbecknell) [reply src/Blazored.Typeahead/BlazoredTypeahead.razor.cs]: I'll push a fix for these whitespace changes.

Comment (zbecknell) [reply src/Blazored.Typeahead/BlazoredTypeahead.razor.cs]: Whew, got it, thanks to here:

https://til.hashrocket.com/posts/696df00135-remove-whitespace-changes-then-git-add-p

**Comment by chrissainty** [comment]
Hey @zbecknell, just wondering why using a CSS selector wouldn't suffice? For example, to make styling changes to the input you could use the following:

```css
.blazored-typeahead__input-multiselect-wrapper > input {
   /*  Styles go here */
}
```

**Comment by zbecknell** [comment]
We have a standard set of CSS from our designer, e.g. `rounded-input` so we can apply that to any given input and match the style.  Just curious what the downside of allowing classes to pass through might be?

We of course `can` target the generated classes, but this option would just add extra flexibility.

**Comment by chrissainty** [comment]
Ok cool, so the reason for my question is that the input is an internal detail of the control and is shown and removed depending on the state of the control. It's also had a lot of its default styling removed.

If you want to influence how the control looks you'd want to apply the styles to the main container div where the `.blazored-typeahead` class is.

**Comment by zbecknell** [comment]
Thanks Chris!  It makes sense why you don't expose that then.  I'll go ahead and close this.

**Pull request closed without merging**